components/inspectit-ocelot-configurationserver/src/main/java/rocks/inspectit/ocelot/security/audit/EntityAuditLogger.java

```diff
@@ -0,0 +1,46 @@
+package rocks.inspectit.ocelot.security.audit;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Component
+@Slf4j
+public class EntityAuditLogger {
+
+    private static final String DEFAULT_USER = "anonymousUser";
```

Comment:
Named it to `anonymousUser` and not `anonymous` as  `anonymousUser` is being used at other places too.